docs: iter 4 (#532)
iterative/terraform-provider-iterative: docs/index.md

TPI is a [Terraform](https://terraform.io) plugin built with machine learning in mind. This CLI tool offers full lifecycle management of computing resources (including GPUs and respawning spot instances) from several cloud vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.

> - **Lower cost with spot recovery**: transparent data backup & auto-recovery from interrupted low-cost spot/preemptible instances

Reviewer: ```suggestion
- **Lower cost with spot recovery**: transparent data checkpoint & restore from interrupted low-cost spot/preemptible instances
```

Reviewer: ```suggestion
- **Lower cost with spot recovery**: transparent data checkpoint/restore & auto-respawning of low-cost spot/preemptible instances
```

Author: ```suggestion
- **Lower cost with spot recovery**: transparent data checkpoint/restore & auto-respawning of low-cost spot/preemptible instances
```

2. **Unified tool for data science and software development teams**:
   TPI provides consistent tooling for both data scientists and DevOps engineers, improving cross-team collaboration. This simplifies compute management to a single config file, and reduces time to deliver ML models into production.
3. **Reproducible, codified environments**:

>    Store hardware requirements & pipelines in a single configuration file with the rest of your ML project code.

Reviewer: ```suggestion
   Store hardware requirements in a single configuration file with the rest of your ML project code.
```

Author: ```suggestion
   Store hardware requirements in a single configuration file alongside the rest of your ML pipeline code.
```
